From: Jiri Pirko <jiri@resnulli.us>
Date: Tue, 12 May 2015 14:56:08 +0200
Subject: flow_dissector: remove unused function flow_get_hlen
 declaration
Patch-mainline: v4.2-rc1
Git-commit: b0a31431b4d81fb1ccc36ce64ce3fe6a0aca4031
References: bsc#922855 FATE#318530

commit 56193d1bce ("net: Add function for parsing the header length out
of linear ethernet frames") added this function declaration but it is
defined nowhere.

Signed-off-by: Jiri Pirko <jiri@resnulli.us>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: David Chang <dchang@suse.com>
---
 include/net/flow_keys.h |    2 --
 1 file changed, 2 deletions(-)

--- a/include/net/flow_keys.h
+++ b/include/net/flow_keys.h
@@ -40,6 +40,4 @@ static inline __be32 skb_flow_get_ports(
 	return __skb_flow_get_ports(skb, thoff, ip_proto, NULL, 0);
 }
 u32 flow_hash_from_keys(struct flow_keys *keys);
-unsigned int flow_get_hlen(const unsigned char *data, unsigned int max_len,
-			   __be16 protocol);
 #endif
